packages/skygear-core/lib/container.js

```diff
@@ -510,6 +528,30 @@ export default class Container extends BaseContainer {
       return this;
     });
   }
+  /**
+   * @private
+   *
+   * Configure container based on the platform.
+   *
+   * This will be called in config function when the platform option change.
+   * Or you can call it after change the platform.
+   * Currently only react native is supported.
+   */
+  configPlatform() {
```

Comment:
`configPlatform` -> `_configPlatform`

I am not sure why you design this function without taking a parameter. I think taking a parameter is better. i.e. `_configPlatform(platform)`. Also, move the `changedPlatform` check to this function?